src/kvm.rs

```diff
@@ -1,6 +1,8 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: AGPL-3.0-only
 
+#![allow(dead_code)]
```

Comment:
If you need this, then you're doing something wrong

Reply:
I would agree, but the compiler was complaining that the variable and function used by KVM wasn't used, despite being used in lib.rs.

Reply:
Obsolete, code has been refactored.